napari/layers/points/points.py

```diff
@@ -1055,14 +1119,12 @@ def interaction_box(self, index) -> np.ndarray:
             starting in the upper-left corner.
         """
         if len(index) == 0:
-            box = None
-        else:
-            data = self._view_data[index]
-            size = self._view_size[index]
-            data = points_to_squares(data, size)
-            box = create_box(data)
+            return None
 
-        return box
+        data = self._view_data[index]
+        size = self._view_size[index]
+        data = points_to_squares(data, size)
+        return create_box(data)
```

Comment:
Generally we prefer to have a single return statement for a function, or have return statements really close together.